webportal/webapp/src/app/account/loginsync/loginsync.component.html

```diff
@@ -0,0 +1,19 @@
+<div class="container">
+  <mat-card>
+    <div>
+      <form class="usa-form signin" (ngSubmit)="onSubmit()">
+        <fieldset class="usa-fieldset">
+          <legend class="usa-legend">Sign in</legend>
+  
+          <label for="username">Username or email address</label>
```

Comment:
This will just be a username, not an email address.

Reply:
kk